docs/configuration.html

```diff
@@ -70,9 +70,13 @@
 
 <h3><a id="consumerconfigs" href="#consumerconfigs">3.3 Consumer Configs</a></h3>
 
-We introduce both the old 0.8 consumer configs and the new consumer configs respectively below.
+We introduce both the new Java consumer configs and the older Scala consumer configs respectively below.
```

Comment:
Maybe even more concisely: "We include both the new Java and the older Scala consumer configs below."